### PR TITLE
feat: Move VerbClient annotation to method

### PR DESCRIPTION
**Pull request by tomdaffurn**
- Move @VerbClient annotation from class to method 
- Rename client method from "call" to \<verbName\>

### CODE REVIEW

File: jvm-runtime/testdata/java/javaclient/src/main/java/xyz/block/ftl/test/TestInvokeGoFromJava.java [hunk 1]
```diff
@@ -59,176 +59,176 @@ public class TestInvokeGoFromJava {
...
             OptionalStringMapVerbClient client) {
-        return client.call(val);
+        return client.optionalStringMapVerb(val);
     }
 
     @Export
     @Verb
     public @Nullable ZonedDateTime optionalTimeVerb(@Nullable ZonedDateTime instant, OptionalTimeVerbClient client) {
-        return client.call(instant);
+        return client.optionalTimeVerb(instant);
```
Comment (stuartwdouglas) [inline jvm-runtime/testdata/java/javaclient/src/main/java/xyz/block/ftl/test/TestInvokeGoFromJava.java:212]: I wonder if we should still stick with 'call' as the default name. You already have the name in the interface, so I don't think we need to duplicate the name.

**Review by stuartwdouglas — APPROVED** [review]

File: jvm-runtime/testdata/java/javaclient/src/main/java/xyz/block/ftl/test/TestInvokeGoFromJava.java (hunk 1 above)
Comment (tomdaffurn) [reply jvm-runtime/testdata/java/javaclient/src/main/java/xyz/block/ftl/test/TestInvokeGoFromJava.java:212]: I was actually wondering if we should combine all verbs into a single interface/class, then we get an API like:
```
    @Export
    @Verb
    public @Nullable ZonedDateTime optionalTimeVerb(@Nullable ZonedDateTime instant, GoModule goModule) {
        return goModule.optionalTimeVerb(instant);
    }
```

Comment (stuartwdouglas) [reply jvm-runtime/testdata/java/javaclient/src/main/java/xyz/block/ftl/test/TestInvokeGoFromJava.java:212]: The issue is then you can't detect usages. I guess if you have a few that are commonly used you could allow a pattern where you have `CombinedInterface extends Client1, Client2`.

Comment (stuartwdouglas) [reply jvm-runtime/testdata/java/javaclient/src/main/java/xyz/block/ftl/test/TestInvokeGoFromJava.java:212]: Let's leave the names as-is for now, and see how it goes.

Comment (tomdaffurn) [reply jvm-runtime/testdata/java/javaclient/src/main/java/xyz/block/ftl/test/TestInvokeGoFromJava.java:212]: Yeah, I hadn't considered the usage detection. Worth discussing all this next week

**Pull request merged**